src/mahoji/lib/abstracted_commands/slayerTaskCommand.ts

```diff
@@ -101,6 +101,9 @@ export async function slayerStatusCommand(mahojiUser: MUser) {
 	const { currentTask, assignedTask, slayerMaster } = await getUsersCurrentSlayerInfo(mahojiUser.id);
 	const { slayer_points: slayerPoints } = mahojiUser.user;
 	const { slayer_task_streak: slayerStreak } = await mahojiUser.fetchStats({ slayer_task_streak: true });
+	const { slayer_wildy_task_streak: wildySlayerStreak } = await mahojiUser.fetchStats({
+		slayer_wildy_task_streak: true
+	});
```

Comment:
Fetch this in the already existing fetchStats call, this will do an extra unnecessary database request.